docker/env/Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV PIP_NO_CACHE_DIR=1
 RUN apt-get install -y --no-install-recommends build-essential cmake libssl-dev zlib1g-dev
 ADD requirements-python.txt .
 RUN mkdir /build && \
-    pip3 install -r requirements-python.txt --root=/build --prefix=./ --ignore-installed --no-warn-script-location
+    pip3 install -r requirements-python.txt --root=/build --prefix=./ --ignore-installed --no-warn-script-location --use-feature=2020-resolver
```

Comment:
I've help tested this feature in europython2020 sprints... it's helpful for our complex requirements.txt